x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/AggregateFunction.java

```diff
@@ -81,6 +81,10 @@ public List<? extends Expression> parameters() {
         return parameters;
     }
 
+    public boolean isConstantFoldable() {
```

Comment:
This is a recent update. For now, I've introduced this one because of the geo aggregations that do not have, from what I've seen, a mechanism for dealing with constants.